xmtp_mls/src/groups/sync.rs

```diff
@@ -517,7 +522,8 @@ where
                     intent_data.message.as_slice(),
                 )?;
 
-                let msg_bytes = msg.tls_serialize_detached()?;
+                let msg_bytes = msg.tls_serialize_detached()
+                    .map_err(|e| GroupError::TlsSerialization(e))?;
```

Comment:
If you have the right `#[from]` on `GroupError` it should magically convert the error types for you with the `?` and you won't need `map_err`